source/testnet/net/guides/run-node-windows.rst

```diff
@@ -0,0 +1,214 @@
+.. include:: ../../variables.rst
+.. _run-node-windows:
+
+================================
+Run and manage a node on Windows
+================================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+This guide describes how organizations can run and manage a node on the Concordium network from a Windows computer. You can also run a node using :ref:`Docker <run-a-node>` or :ref:`Ubuntu <run-node-ubuntu>`.
+
+.. Note::
+      Currently, you can only run a Windows node on the testnet.
+
+Prequisites
+===========
+
+-  Run Windows 10 or later or Windows Server 2019 or later.
+-  Meet the :ref:`minimum system requirements<requirements-run-node>` for running a node.
+
+Install and run a node
+======================
+
+#. Go to :ref:`Downloads<downloads>`, and download the latest Windows Installer package (.msi file).
+
+#. In the folder where you downloaded the .msi file, double-click the .msi file. The **Concordium Node Setup Wizard** opens. Select **Next**. If you see a message saying *Windows protected your PC*, select **More info**, and then select **Run anyway**.
+
+#. Accept the terms of the End-User License Agreement and select **Next**.
+
+#. Set the **Destination** folders.
+
+   - In **Install Concordium Node to**, specify where to store the executable code.
+
+   - In **Install Concordium Node configuration and data to**, specify where to store the node configuration and the blockchain databases and logs. Choose a drive that has sufficient space to accommodate the database and is sufficiently fast. For more information, see :ref:`Requirements for running a node<requirements-run-node>`.
+
+     .. image:: ../images/run-node/Node-setup-win-1.png
+        :width: 65%
+
+#. Select **Next** and configure the **Node Runner Service**. You have the following options:
+
+   - **Automatically start nodes at system startup**:  When selected, the node runs when the system starts. Choose this option when you plan to use the node frequently and need it to be up-to-date at short notice. If you don’t select this option, you’ll have to start the node manually when required, for example, when you want to use the Desktop Wallet or concordium-client. If you choose to start the node manually, it might take longer for the node to get up-to-date with the blockchain depending on when the node was last up-to-date.
+
+   - **Start nodes after installation is complete**: The node will automatically start running after the installation is complete. Do not choose this option if you want to make further configuration changes before starting the node.
+
+   - **Mainnet** or **Testnet**: Select **run a testnet node**.
```

Comment:
Since this is not an either-or choice, they need to deselect "run a mainnet node" if they only want a testnet node. It might be useful to have a bit more explanation of what the options do.